sounddevice.py

```diff
@@ -587,6 +587,38 @@ def get_status():
         raise RuntimeError('play()/rec()/playrec() was not called yet')
 
 
+def get_stream():
+    """Get a reference to the current Stream.
+
+    This applies only to Streams created by calls to `play()`, `rec()`,
+    or `playrec()`.
+
+    Returns
+    -------
+    Stream
+        An `OutputStream`, `InputStream`, or `Stream` associated with
+        the last invocation of `play()`, `rec()` or `playrec()`
+        respectively.
+
+    Examples
+    --------
+
+    >>> import sounddevice as sd
+    >>> from time import sleep
+    >>> sd.play('test.wav')
+    >>> print('Playing test.wav...', end='')
+    >>> stream = sd.get_stream()
+    >>> while stream.active:
+    >>>     sleep(1.0)
+    >>>     print('.', end='')
+    >>> print(' Done.')
+    """
```

Comment:
For consistency, the docstring should end with a blank line.
I think this is the NumPy documentation style ...